### DISCUSSION TITLE
Weather source on the top

### DISCUSSION THREAD

**Discussion by ghost** [breezy-weather/breezy-weather#1148] [Ideas]
### Describe your idea

Hello,
I use multiple sources for the same city.
When I navigate from left to right, I scroll down to read the source name, it is not practical...

Can you add an option to display the source under the city name at the top please (or an icon)

Lyon (MétéoFrance)
Lyon (OpenMéteo)
Lyon (OpenWeather)

Thanks :)

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.4 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Breezy Weather is not made for comparison. These sources are provided for free and we ask for fair use, so please stick to one. Use something else like radars for comparison.

For a city in France, it's even more useless as Météo-France provides the data for Open-Meteo and OpenWeather.

Finally, the source is a combination of many sources, such as a different air quality source. Some sources require a specific credit message and we won't display twice, once with the specific message, and the other with just all source names comma-separated, which wouldn't fit anyway (maybe with short city names like Lyon + only one source used, but that would be the rare exception).

**Comment by papjul** [comment]
You might be interested in #752, although I still maintain that it makes no sense to add multiple times the same city ;)
